buzzmobile/process/gps_mapper/gps_mapper.py

```diff
@@ -39,8 +44,17 @@ def update_image():
         width = int(round(frames['x_range'] * x_scale))
         point = (frames['location'][0], -frames['location'][1])
         if frames['y_range'] is not 0:
-            point = interpolate.normalize_single_point(frames['y_range'], frames['x_range'], height, width, top_left, bottom_right, point)
-        result = interpolate.xwindow(frames['points'], point, frames['bearing'], rospy.get_param('image_height'), rospy.get_param('image_width'))
+            point = interpolate.normalize_single_point(frames['y_range'],
+                                                       frames['x_range'],
+                                                       height, width,
+                                                       top_left, bottom_right,
+                                                       point)
+        result = interpolate.xwindow(frames['points'], point, frames['bearing'],
+                                     line_width,
+                                     rospy.get_param('sigma_x'),
+                                     rospy.get_param('sigma_y'),
+                                     rospy.get_param('image_height'),
+                                     rospy.get_param('image_width'))
```

Comment:
Don't call `get_param` on every `update_image()`. Save the params as global values on the top of the file.